src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1234,48 +1236,68 @@ public static string GetDecoratedSymbolName(SymbolReference symbolReference)
             await requestContext.SendResult(result);
         }
 
+        // Since the NamedPipeConnectionInfo type is only available in 5.1+
+        // we have to use Activator to support older version of PS.
+        // This code only lives in the v1.X of the extension.
+        // The 2.x version of the code can be found here:
+        // https://github.com/PowerShell/PowerShellEditorServices/pull/881
+        private static Type _namedPipeConnectionInfoType = Type.GetType("System.Management.Automation.Runspaces.NamedPipeConnectionInfo, System.Management.Automation");
```

Comment:
The type is only used to get the constructor so it doesn't need to be stored.

Also you can skip assembly resolution by referencing a type from the assembly, e.g. `typeof(PSObject).Assembly.GetType("System.Management.Automation.Runspaces.NamedPipeConnectionInfo")`

Reply:
`typeof(PSObject).Assembly` not in .NET Standard 1.6 :(

Reply:
`typeof(PSObject).GetTypeInfo().Assembly` worked. Changed.